docs/LDAP.rst

```diff
@@ -21,61 +21,287 @@ plug to a directory.
+   |                                                | alias for com.sun.jndi.ldap.connect.pool.maxsize        |                           |                                    |                            |
+   +------------------------------------------------+---------------------------------------------------------+---------------------------+------------------------------------+----------------------------+
+   | exo.ldap.connection.pool.timeout               | Number of milliseconds that an idle connection          | Any non negative number   |                                    |                            |
+   |                                                | may remain in the pool without being closed             |                           |                                    |                            |
+   |                                                | and removed from the pool,                              |                           |                                    |                            |
+   |                                                | alias for com.sun.jndi.ldap.connect.pool.timeout        |                           |                                    |                            |
+   +------------------------------------------------+---------------------------------------------------------+---------------------------+------------------------------------+----------------------------+
+   | exo.ldap.connection.pool.protocol              | List of space-separated protocol types of               | ``plain`` and/or ``ssl``  | ``plain ssl``                      |                            |
+   |                                                | connections that may be pooled,                         |                           |                                    |                            |
+   |                                                | alias for com.sun.jndi.ldap.connect.pool.protocol       |                           |                                    |                            |
+   +------------------------------------------------+---------------------------------------------------------+---------------------------+------------------------------------+----------------------------+
+   | exo.ldap.search.timelimit                      | Number of milliseconds that a search may last           | Any positive number       | 10000                              |                            |
+   +------------------------------------------------+---------------------------------------------------------+---------------------------+------------------------------------+----------------------------+
+   | exo.ldap.users.base.dn                         | Semicolon-separated list of full DNs                    |                           | ``ou=users,dc=company,dc=org``     |                            |
```

Comment:
can should we put an example with more than one value ?

Reply:
Yes, done